pkg/client/docker/image.go

```diff
@@ -34,5 +36,16 @@ func PullImage(ctx context.Context, image string) error {
 	// Docker run will put the pull logs in stderr, but docker pull will put them in stdout.
 	// We discard them here, so they don't spam the user. They'll get errors through stderr if it comes to it.
 	cmd.Stdout = io.Discard
-	return cmd.Run()
+
+	// Only print stderr if the return code is non-zero
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
+
+	err = cmd.Run()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, stderr.String())
```

Comment:
```suggestion
		fmt.Fprintln(os.Stderr, stderr.String())
```
Are you sure you need the ln though ?